### ISSUE TITLE
Accessibility: jupyter.org

### ISSUE THREAD

**Issue by ade-ronke** [jupyterhub/outreachy#53]
### What page is this for?

http://jupyter.org/

### WAVE accessibility report

![Jupyter org](https://user-images.githubusercontent.com/86165428/195857714-df29ed6f-f41d-41df-8c20-8d0908ef4897.PNG)

### The HTML element to be changed

```html
Based on the assessment, the following have been spotted and can be improved using the web content accessibility guidelines (WCAG):
1.Contrast Error: The view buttons have contrast issues between text and background colors. The text and background color have a less contrast ratio of 3.19:1 which is less than 4.5:1, or large text (larger than 18 points or 14 points bold) and this makes it uneasy for users with low vision to navigate the website.

2.Alerts: 
Alternative text : The two logos have the same alternative text, this often causes redundancy or indicates incorrect alternative text. Due to this, users who make use of screen readers will not get adequate information.

3.Heading: Headings communicate the organization of the content on the page. Web browsers, plug-ins, and assistive technologies like screen readers use them to provide in-page navigation. Screen readers typically offer a shortcut that brings up a list of all headings within the page as a means of user navigation. If screen readers announce the heading level, the skipped number would make it seem like the user had missed some hidden content when they haven't. Suggestion: A hierarchy of header elements should be used to solve this. 

Missing first level heading element (1) : A page does not have an <h1> element. Headings facilitate page navigation for users of many assistive technologies. They also provide semantic and visual meaning and structure to the document. A first level heading (<h1>) should be present on nearly all pages. It should contain the most important heading on the page (generally the document title). If the page presents a main heading, place it within an <h1> element. Add other sub-headings as necessary.


5.Navigation: It means that a <nav> element or navigation landmark is present.Two Navigation Icons is present on the page. A navigation icon should not be present in the footer as it is to be found in the page header.
```

### Your proposed HTML (or CSS), after change

```html
Suggestion for Contrast error: It can be fixed by reducing the brightness of the background color. A contrast ratio 8.25:1 would be better for users, especially those with low vision. (Background color= #823912 , foreground color= #FFFFFF). We can also get color suggestions in the Contrast option of the WAVE tool.

Suggestion for Alternative text: Ensure that the alternative text for the image or image input provides a succinct, yet an equivalent alternative to the content and function of the image. Screen readers and browser presentation inform the user that the object is an image, so the alternative text of "image of..." (and similar) should be avoided. If the image does not convey content or if the content is presented in nearby text (e.g., a caption), null/empty alternative text (alt="") is appropriate.

Suggestion for Heading: A <p></> tag could be used instead of the heading elements used as the text is not a section heading.

Suggestion for Navigation: Navigation links should be embedded in the Nav bars as ordered lists in the footer. The user should be able to navigate when ordered lists are hovered or clicked on.
```

**Comment by ade-ronke** [comment]
1.Contrast.
**Before**
HTML
<a class="orange-button " href="/install" style="color: rgb(255, 255, 255); background-color: rgb(228, 110, 46 )>Try it in your browser</a>
CSS
.orange-button{ 
  Font-size:1.2em;
  Border-radius:2px;
  Text-shadow: 0px 0px 4px #000;
  Padding:0.75em 1.25em;
 Margin:0 0.5em 0 0;
 Text-align:center;
 Text-decoration:none;  
}

**AFTER**
HTML
<a class=”orange-button” href=”/try”> Try it in your browser</a>
CSS
.orange-button{ 
   Backround-color: rgb (130,57 ,18);
   Color:rgb(255,255,255);
  Font-size:1.2em;
  Border-radius:2px;
 Text-shadow:0px 0px 4px #000;
  Padding:0.75em 1.25em;
 Margin:0 0.5em 0 0;
 Text-align: center;
 Text-decoration: none;  
}
2. Alerts: 
Before
<span>
  <img class =”img-responsive fade-in three”  src=”/assets/homepage/circle2.svg” width=”677” height=”286” alt=”circle of programming language icons” loading=”eager”>
</span>
AFTER
<span>
  <img class =”img-responsive fade-in three”  src=”/assets/homepage/circle2.svg” width=”677” height=”286” alt=” programming language icons” loading=”eager”>
</span>
4.Missing first level heading element (1)
BEFORE
<p class=”dropshadow”>  Cité des sciences, Paris</p>
<p class=”dropshadow”> CFP Opens October 31</p>

AFTER
<h2 class=”dropshadow”> Cité des sciences, Paris </h2>
<h3 class=”dropshadow”> CFP Opens October 31</h3>

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
